secant/Features/Backup Flow/RecoveryPhraseDisplayStore.swift

```diff
@@ -0,0 +1,86 @@
+//
+//  RecoveryPhraseDisplayStore.swift
+//  secant-testnet
+//
+//  Created by Francisco Gindre on 10/26/21.
+//
+
+import Foundation
```

Comment:
This Feature is in a folder named Backup Flow, with a space between the words.  For scripting purposes, or CI purposes, you may want to consider not including that space, as different operating systems or environments will treat that differently.  I've seen this cause issues a few times before, and I believe it's considered a best practice to not include spaces in folder names for this reason.

Reply:
Yes this is a good Idea.